exporter/datadogexporter/translate_traces.go

```diff
@@ -487,7 +487,14 @@ func decodeAPMId(id string) uint64 {
 	return val
 }
 
-func getDatadogSpanName(s pdata.Span, datadogTags map[string]string) string {
+func getDatadogSpanName(s pdata.Span, datadogTags map[string]string, spanNameAsResourceName bool) string {
+	// Option created to maintain similarity with the OpenTelemetry semantic conventions
+	// https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions
+	// https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1909
```

Comment:
Can you add these comments to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/example/config.yaml instead?

Reply:
I've left the comments there but added them to the config example.